.github/dependabot.yml

```diff
@@ -51,6 +51,11 @@ updates:
       # Starting with 6.x, Spring Security requires Java 17 at a minimum.
       - dependency-name: "org.springframework.security:spring-security-bom"
         versions: [">=6.0.0"]
+
+      # Starting with 7.x, Guice switches from javax.* to jakarta.* bindings.
+      # See https://github.com/google/guice/wiki/Guice700
+      - dependency-name: "com.google.inject:guice-bom"
+        versions: [">=6.0.0"]
```

Comment:
Why is this ignoring versions greater than or equal to 6.0.0 when some of those versions are compatible, as evidenced by the fact that this PR is itself upgrading to version 6.0.0. The only versions that should be ignored are the ones that are incompatible; namely, those greater than or equal to 7.0.0.